File: tests/fixtures/phase3/positive/Surfaces.cs
using System.Net.Http;

class Surfaces
{
    void Review(string query, string url, string path, string code, HttpClient client)
    {
        var command = new SqlCommand(query);
        client.GetAsync(url);
        File.ReadAllText(path);
        File.WriteAllText(path, "content");
        CSharpScript.EvaluateAsync(code);
    }
}

File: tests/fixtures/phase3/positive/surfaces.go
package fixture

import (
	"database/sql"
	"net/http"
	"os"
	"plugin"
)

func review(db *sql.DB, query string, url string, path string) {
	db.Query(query)
	http.Get(url)
	os.Open(path)
	os.Create(path)
	plugin.Open(path)
}

